### DISCUSSION TITLE
Patch bundled libcurl when upstream CVEs come out

### DISCUSSION THREAD

**Discussion by DemiMarie** [sagebind/isahc#171] [General]
Many of them may not effect us (TFTP, FTP, etc), but we should still patch them.

**Comment by sagebind** [comment]
Any particular examples you are referring to?

This is probably an issue for the curl-sys crate to handle, as long as we keep up to date with the latest curl-sys versions. To my knowledge, curl-sys keeps relatively up-to-date with upstream source.

**Comment by DemiMarie** [comment]
None in particular; cURL’s HTTP implementation has a fantastic security record.

**Comment by sagebind** [comment]
The curl and curl-sys crate have a pretty good record of staying up-to-date with upstream libcurl patches. Feel free to let me know if there's more work to be done.
